src/transformers/modeling_layoutlm.py

```diff
@@ -0,0 +1,905 @@
+# coding=utf-8
+# Copyright 2018 The Microsoft Research Asia LayoutLM Team Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" PyTorch LayoutLM model. """
+
+
+import logging
+import math
+
+import torch
+from torch import nn
+from torch.nn import CrossEntropyLoss
+
+from .activations import gelu, gelu_new, swish
+from .configuration_layoutlm import LayoutLMConfig
+from .file_utils import add_code_sample_docstrings, add_start_docstrings, add_start_docstrings_to_callable
+from .modeling_outputs import BaseModelOutput, BaseModelOutputWithPooling, MaskedLMOutput, TokenClassifierOutput
+from .modeling_utils import (
+    PreTrainedModel,
+    apply_chunking_to_forward,
+    find_pruneable_heads_and_indices,
+    prune_linear_layer,
+)
+
+
+logger = logging.getLogger(__name__)
+
+_CONFIG_FOR_DOC = "LayoutLMConfig"
+_TOKENIZER_FOR_DOC = "LayoutLMTokenizer"
+
+LAYOUTLM_PRETRAINED_MODEL_ARCHIVE_LIST = [
+    "layoutlm-base-uncased",
+    "layoutlm-large-uncased",
+]
+
+
+def mish(x):
```

Comment:
mish can be added to the `activations` module and `ACT2FN` imported from  there I think.